### PR TITLE
Update README for .serve and update language

### PR DESCRIPTION
**Pull request by discdiver**
### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`, `docs`.
  

For documentation changes:

- [ ] This pull request includes redirect settings in `netlify.toml` for files that are removed or renamed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,15 +20,33 @@
 
 # Prefect
 
-Prefect is an orchestrator for data-intensive workflows. It's the simplest way to transform any Python function into a unit of work that can be observed and orchestrated. With Prefect, you can build resilient, dynamic workflows that react to the world around them and recover from unexpected changes. With just a few decorators, Prefect supercharges your code with features like automatic retries, distributed execution, scheduling, caching, and much more. Every activity is tracked and can be monitored with the Prefect server or Prefect Cloud dashboard.
+Prefect is a workflow application framework. It's the simplest way to transform Python code into an interactive workflow application that can be observed and orchestrated.
+Prefect sits at the center of the many systems you interact with every day.
+It provides you an API middleware that makes it possible to understand and react to event.
```
[the remark this answers is not in the document]
Comment (billpalombi) [reply README.md:25]: The use of the term "middleware" feels a bit off to me. I think we may need to elaborate on how workflows can act as middleware it to make it clear what we mean. That topic could merit an entire blog post.

File: README.md [hunk 2]
```diff
@@ -20,15 +20,33 @@
 
 # Prefect
 
-Prefect is an orchestrator for data-intensive workflows. It's the simplest way to transform any Python function into a unit of work that can be observed and orchestrated. With Prefect, you can build resilient, dynamic workflows that react to the world around them and recover from unexpected changes. With just a few decorators, Prefect supercharges your code with features like automatic retries, distributed execution, scheduling, caching, and much more. Every activity is tracked and can be monitored with the Prefect server or Prefect Cloud dashboard.
+Prefect is a workflow application framework. It's the simplest way to deploy Python code as an interactive workflow application that can be observed and orchestrated.
+Prefect sits at the center of the many systems you interact with every day.
+It provides you an API that makes it possible to understand and react to events.
+Prefect also allows you to standardize your workflows across your organization.
```
Comment (billpalombi) [inline README.md:26]:
```suggestion
Prefect also allows you to standardize workflows across your organization.
```

File: README.md [hunk 3]
```diff
@@ -42,41 +60,44 @@ def github_stars(repos: List[str]):
...
-
-```bash
-pip install prefect
+```python
+    github_stars.serve(name="first-deployment", cron="* * * * *")
 ```
 
-Start by exploring the [core concepts of Prefect workflows](https://docs.prefect.io/concepts/), then follow one of our [friendly tutorials](https://docs.prefect.io/tutorials/first-steps) to learn by example.
+You now have a server running locally that is looking for scheduled deployments!
+Additionally you can run your workflow manually from the UI or CLI - and if you're using Prefect Cloud, you can even run deployments in response to an [events](https://docs.prefect.io/latest/concepts/automations/).
```
Comment (billpalombi) [inline README.md:82]:
```suggestion
Additionally you can run your workflow manually from the UI or CLI - and if you're using Prefect Cloud, you can even run deployments in response to [events](https://docs.prefect.io/latest/concepts/automations/).
```

File: README.md [hunk 4]
```diff
@@ -20,15 +20,33 @@
 
 # Prefect
 
-Prefect is an orchestrator for data-intensive workflows. It's the simplest way to transform any Python function into a unit of work that can be observed and orchestrated. With Prefect, you can build resilient, dynamic workflows that react to the world around them and recover from unexpected changes. With just a few decorators, Prefect supercharges your code with features like automatic retries, distributed execution, scheduling, caching, and much more. Every activity is tracked and can be monitored with the Prefect server or Prefect Cloud dashboard.
+Prefect is a workflow application framework. It's the simplest way to deploy Python code as an interactive workflow application that can be observed and orchestrated.
+Prefect sits at the center of the many systems you interact with every day.
+It provides you an API that makes it possible to understand and react to events.
```
Comment (billpalombi) [inline README.md:25]: I see how the API supports reacting to events, but I don't see how it supports understanding events.

File: README.md [hunk 5]
```diff
@@ -20,15 +20,33 @@
 
 # Prefect
 
-Prefect is an orchestrator for data-intensive workflows. It's the simplest way to transform any Python function into a unit of work that can be observed and orchestrated. With Prefect, you can build resilient, dynamic workflows that react to the world around them and recover from unexpected changes. With just a few decorators, Prefect supercharges your code with features like automatic retries, distributed execution, scheduling, caching, and much more. Every activity is tracked and can be monitored with the Prefect server or Prefect Cloud dashboard.
+Prefect is a workflow application framework. It's the simplest way to deploy Python code as an interactive workflow application that can be observed and orchestrated.
```
Comment (sarahmk125) [inline README.md:23]: I would not go so hard on "workflow application framework".
```suggestion
Prefect is a developer platform for building, observing, and triaging workflows. Powered by a Python library, it pairs with a responsive dashboard and observability for getting a complete picture of your workflows.
```

File: README.md (hunk 4 above)
Comment (sarahmk125) [inline README.md:25]:
```suggestion
Prefect allows you to expose your workflows through an API so teams dependent on you can programmatically access your pipelines, business logic, and more.
```

File: README.md (hunk 2 above)
Comment (sarahmk125) [reply README.md:26]:
```suggestion
Prefect also allows you to standardize workflow development and deployment across your organization.
```

File: README.md [hunk 6]
```diff
@@ -20,15 +20,33 @@
...
+Prefect sits at the center of the many systems you interact with every day.
+It provides you an API that makes it possible to understand and react to events.
+Prefect also allows you to standardize your workflows across your organization.
+
+With Prefect, you can build resilient, dynamic workflows that react to the world around them and recover from unexpected changes.
+With just a few decorators, Prefect supercharges your code with features like automatic retries, distributed execution, scheduling, caching, and much more.
+
+Every activity is tracked and can be monitored with a self-hosted [Prefect server](https://docs.prefect.io/latest/guides/host/) instance or managed [Prefect Cloud](https://docs.prefect.io/latest/cloud/) dashboard.
```
Comment (sarahmk125) [inline README.md:31]:
```suggestion
Every activity is tracked and can be monitored with a self-hosted [Prefect server](https://docs.prefect.io/latest/guides/host/) instance or managed [Prefect Cloud](https://www.prefect.io/cloud-vs-oss?utm_source=oss&utm_medium=oss&utm_campaign=oss_gh_repo&utm_term=none&utm_content=none) dashboard.
```

File: README.md [hunk 7]
```diff
@@ -42,41 +60,44 @@ def github_stars(repos: List[str]):
...
 
 ## Join the community
 
-Prefect is made possible by the fastest growing community of thousands of friendly data engineers. Join us in building a new kind of workflow system. The [Prefect Slack community](https://prefect.io/slack) is a fantastic place to learn more about Prefect, ask questions, or get help with workflow design. The [Prefect Discourse](https://discourse.prefect.io/) is a community-driven knowledge base to find answers to your Prefect-related questions. All community forums, including code contributions, issue discussions, and slack messages are subject to our [Code of Conduct](https://discourse.prefect.io/faq).
+Prefect is made possible by the fastest growing community of thousands of friendly data engineers. Join us in building a new kind of workflow application system. The [Prefect Slack community](https://prefect.io/slack) is a fantastic place to learn more about Prefect, ask questions, or get help with workflow design. All community forums, including code contributions, issue discussions, and slack messages are subject to our [Code of Conduct](https://discourse.prefect.io/faq).
```
Comment (sarahmk125) [inline README.md:100]:
```suggestion
Prefect is made possible by the fastest growing community of thousands of friendly data engineers. Join us in building a new kind of workflow system, inspired by application development. The [Prefect Slack community](https://prefect.io/slack) is a fantastic place to learn more about Prefect, ask questions, or get help with workflow design. All community forums, including code contributions, issue discussions, and slack messages are subject to our [Code of Conduct](https://discourse.prefect.io/faq).
```

File: README.md [hunk 8]
```diff
@@ -20,15 +20,33 @@
 
 # Prefect
 
-Prefect is an orchestrator for data-intensive workflows. It's the simplest way to transform any Python function into a unit of work that can be observed and orchestrated. With Prefect, you can build resilient, dynamic workflows that react to the world around them and recover from unexpected changes. With just a few decorators, Prefect supercharges your code with features like automatic retries, distributed execution, scheduling, caching, and much more. Every activity is tracked and can be monitored with the Prefect server or Prefect Cloud dashboard.
+Prefect is a workflow application framework. It's the simplest way to transform Python code into an interactive workflow application that can be observed and orchestrated.
```
[the remark this answers is not in the document]
Comment (billpalombi) [reply README.md]: Thats probably a good idea

**Comment by discdiver** [comment]
Thanks for the reviews! Will hold for now.

**Comment by csreuter** [comment]
I will propose some language next week, but let's not make any changes yet please.

File: README.md [hunk 9]
```diff
@@ -42,41 +60,44 @@ def github_stars(repos: List[str]):
...
 
-Read more about Prefect Cloud [here](https://www.prefect.io/cloud-vs-oss?utm_source=oss&utm_medium=oss&utm_campaign=oss_gh_repo&utm_term=none&utm_content=none) or signup to [try it for yourself](https://app.prefect.cloud?utm_source=oss&utm_medium=oss&utm_campaign=oss_gh_repo&utm_term=none&utm_content=none).
+Read more about Prefect Cloud [here](https://www.prefect.io/cloud-vs-oss?utm_source=oss&utm_medium=oss&utm_campaign=oss_gh_repo&utm_term=none&utm_content=none) or sign up to [try it for yourself](https://app.prefect.cloud?utm_source=oss&utm_medium=oss&utm_campaign=oss_gh_repo&utm_term=none&utm_content=none).
 
 ![Prefect Automations](/docs/img/ui/automations.png)
 
+## Next steps
+
+There's lots more you can do to orchestrate and observe your workflows with Prefect!
+Start with our [friendly tutorial](https://docs.prefect.io/tutorials/first-steps) or explore the [core concepts of Prefect workflows](https://docs.prefect.io/concepts/).
```
Comment (discdiver) [inline README.md:96]:
```suggestion
Start with our [friendly tutorial](https://docs.prefect.io/tutorials) or explore the [core concepts of Prefect workflows](https://docs.prefect.io/concepts/).
```

File: README.md [hunk 10]
```diff
@@ -20,15 +20,33 @@
 
 # Prefect
 
-Prefect is an orchestrator for data-intensive workflows. It's the simplest way to transform any Python function into a unit of work that can be observed and orchestrated. With Prefect, you can build resilient, dynamic workflows that react to the world around them and recover from unexpected changes. With just a few decorators, Prefect supercharges your code with features like automatic retries, distributed execution, scheduling, caching, and much more. Every activity is tracked and can be monitored with the Prefect server or Prefect Cloud dashboard.
+Prefect is a workflow application framework. It's the simplest way to transform Python code into an interactive workflow application that can be observed and orchestrated.
+Prefect sits at the center of the many systems you interact with every day.
+It provides you an API middleware that makes it possible to understand and react to event.
+Prefect also allows you to standardize your workflows across your organization.
+
+With Prefect, you can build resilient, dynamic workflows that react to the world around them and recover from unexpected changes.
```
Comment (csreuter) [inline README.md:28]: I really like this @discdiver

File: README.md (hunk 8 above)
Comment (csreuter) [inline README.md]: @discdiver  How about "Prefect is an orchestration and observability platform for building, observing, and triaging workflows. It's the simplest way to transform Python code into an interactive workflow application."

File: README.md (hunk 5 above)
Comment (billpalombi) [reply README.md:23]: While I'm comfortable emphasizing the pythonic nature of the SDK, I don't think "Powered by a Python library" is particularly accurate or informative here.

File: README.md [hunk 11]
```diff
@@ -20,15 +20,35 @@
 
 # Prefect
 
-Prefect is an orchestrator for data-intensive workflows. It's the simplest way to transform any Python function into a unit of work that can be observed and orchestrated. With Prefect, you can build resilient, dynamic workflows that react to the world around them and recover from unexpected changes. With just a few decorators, Prefect supercharges your code with features like automatic retries, distributed execution, scheduling, caching, and much more. Every activity is tracked and can be monitored with the Prefect server or Prefect Cloud dashboard.
+Prefect is an orchestration and observability platform for building, observing, and triaging workflows. 
+It's the simplest way to transform Python code into an interactive workflow application.
+
+Prefect sits at the center of the many systems you interact with every day.
```
Comment (billpalombi) [inline README.md:26]:
```suggestion
```

Comment (billpalombi) [reply README.md:26]: I don't think this is true.

File: README.md [hunk 12]
```diff
@@ -42,41 +62,44 @@ def github_stars(repos: List[str]):
...
 
 ## Join the community
 
-Prefect is made possible by the fastest growing community of thousands of friendly data engineers. Join us in building a new kind of workflow system. The [Prefect Slack community](https://prefect.io/slack) is a fantastic place to learn more about Prefect, ask questions, or get help with workflow design. The [Prefect Discourse](https://discourse.prefect.io/) is a community-driven knowledge base to find answers to your Prefect-related questions. All community forums, including code contributions, issue discussions, and slack messages are subject to our [Code of Conduct](https://discourse.prefect.io/faq).
+Prefect is made possible by the fastest growing community of thousands of friendly data engineers. Join us in building a new kind of workflow system, inspired by application development. The [Prefect Slack community](https://prefect.io/slack) is a fantastic place to learn more about Prefect, ask questions, or get help with workflow design. All community forums, including code contributions, issue discussions, and slack messages are subject to our [Code of Conduct](https://discourse.prefect.io/faq).
```
Comment (billpalombi) [inline README.md:102]:
```suggestion
Prefect is made possible by the fastest growing community of thousands of friendly data engineers. Join us in building a new kind of workflow system. The [Prefect Slack community](https://prefect.io/slack) is a fantastic place to learn more about Prefect, ask questions, or get help with workflow design. All community forums, including code contributions, issue discussions, and slack messages are subject to our [Code of Conduct](https://discourse.prefect.io/faq).
```

Comment (billpalombi) [reply README.md:102]: I wouldn't emphasize this but that's a weakly held preference.

**Review by billpalombi — APPROVED** [review]
lgtm

**Pull request merged**